docs/decisions/0002-deploy-pipelines.md

```diff
@@ -0,0 +1,53 @@
+---
+status: accepted
+date: 2022-11-04
+---
+# Deploy pipeline along with the operator
+
+## Context and Problem Statement
+
+CNBi feature include OpenShift Pipelines, that are used for task actions.<br>
+The pipelines are not entirely dependent on the CNBi Operator. <br>
+However the Operator trigger the pipelines execution, so there is a dependency 
+from Operator end.
```

Comment:
```suggestion
CNBi feature include OpenShift Pipelines, that are used for task actions.
The pipelines are not entirely dependent on the CNBi Operator.
However the Operator trigger the pipelines execution, so there is a dependency from Operator end.
```
I think the mardown expect one phrase per line without line length limiting

Reply:
thanks :+1: